CONTRIBUTING.md

```diff
@@ -0,0 +1,28 @@
+# Contribution Guide
+
+The infrastructure project contains:
+
+1. The [Ansible](ansible) scripts for bootstrapping the build and test hosts (including a way to test Ansible)
+1. The overriding [Documentation](docs) for the build farm
+
+To make a change please:
+ 
+1. Fork this repository
+1. Create a branch off your fork
+1. Make the change
+1. Test it (see below)
+1. Submit a Pull Request
+
+Only reviewers in the `admin_infrastructure` team have permission to merge requests for this `openjdk-infrastructure` repo, 
```

Comment:
Might be worth showing a link to the reviewers, it is possible to create a reviewers group on GitHub, and link to it?